services/net/reporting/ReportingManager.cs

```diff
@@ -974,7 +974,7 @@ private async Task<Dictionary<EmailSentTo, List<UserEmail>>> GetEmailAddressesAs
             if (user.User == null) throw new InvalidOperationException("Report subscriber is missing user information");
 
             // Only include users who have not received an email yet.
-            var (a, b, c) = await GetEmailAddressesAsync(user.UserId, user.User.GetEmail(), user.User.AccountType, user.SendTo);
+            var (a, b, c) = await GetEmailAddressesAsync(user.UserId, user.User.GetEmail(), user.User.IsVacationMode(), user.User.AccountType, user.SendTo);
```

Comment:
This is not needed.  You already filtered out users on vacation before calling this method.